commands/init.ts

```diff
@@ -53,13 +53,16 @@ const TRACKING_STATUS = {
   COMPLETE: 'complete',
 };
 
-const personalAccessKeyConfigCreationFlow = async (env, account) => {
+async function personalAccessKeyConfigCreationFlow(
+  env: Environment,
+  account?: number
+) {
```

Comment:
Missing return type here + in a few other functions in this file!

Reply:
Done in [be71b49](https://github.com/HubSpot/hubspot-cli/pull/1401/commits/be71b495f9f24049001fda5d7c3328c7479fcdc8)